style/tailwind.css

```diff
@@ -1,141 +1,264 @@
-  @apply list-inside list-disc;
-}
+    36% {
+      color: var(--color-emerald-400);
+    }
 
-.markdown-content ol {
-  @apply list-inside list-decimal;
-}
+    42% {
+      color: var(--color-teal-400);
+    }
 
-.markdown-content li {
-  @apply my-1.5;
-}
+    48% {
+      color: var(--color-cyan-400);
+    }
 
-.markdown-content a {
-  @apply text-accent font-medium;
-}
+    54% {
+      color: var(--color-sky-400);
+    }
 
-.markdown-content a:hover {
-  @apply underline underline-offset-2;
-}
+    60% {
+      color: var(--color-blue-400);
+    }
 
-.markdown-content img {
-  max-height: 40vh;
-  @apply max-w-full h-auto;
-}
+    66% {
+      color: var(--color-indigo-400);
+    }
 
-.markdown-content img[title^="emoji"] {
-  @apply inline w-16;
-}
+    72% {
+      color: var(--color-violet-400);
+    }
 
-.markdown-content hr {
-  @apply border-secondary;
-}
+    78% {
+      color: var(--color-purple-400);
+    }
 
-.markdown-content table {
-  @apply table w-auto mx-auto shadow-lg rounded-md bg-base-100;
-}
+    84% {
+      color: var(--color-fuchsia-400);
+    }
 
-.markdown-content thead tr {
-  @apply bg-base-300;
-}
+    90% {
+      color: var(--color-pink-400);
+    }
 
-.markdown-content tbody tr {
-  @apply border-t-2 border-accent;
+    95% {
+      color: var(--color-rose-400);
+    }
+  }
 }
 
-.markdown-content tbody td:not(:first-child) {
-  @apply border-accent border-l-2;
-}
+.grid-areas-post-listing {
+  grid-template-areas:
+    "title title thumbnail"
+    "to to thumbnail"
+    "vote actions thumbnail";
+  grid-template-rows: 1fr 1fr 1fr;
+  grid-template-columns: min-content auto min-content;
 
-.markdown-content thead th:not(:first-child) {
-  @apply border-accent border-l-2;
+  @variant sm {
+    grid-template-areas:
+      "vote thumbnail title"
+      "vote thumbnail to"
+      "vote thumbnail actions";
+    grid-template-rows: 1fr 2fr 1fr;
+    grid-template-columns: min-content min-content auto;
+  }
 }
 
-.markdown-content summary {
-  @apply flex justify-start cursor-pointer break-all;
+.grid-in-vote {
+  grid-area: vote;
 }
 
-.markdown-content summary::before {
-  content: " ";
-  mask-image: url("/icons.svg#css-warning");
-  -webkit-mask-image: url("/icons.svg#css-warning");
-  @apply bg-warning-content block size-6 min-w-6 min-h-6 me-1;
+.grid-in-title {
+  grid-area: title;
 }
 
-.markdown-content summary::after {
-  content: " ";
-  mask-image: url("/icons.svg#css-caret");
-  -webkit-mask-image: url("/icons.svg#css-caret");
-  @apply bg-warning-content block size-6 min-w-6 min-h-6 ms-auto transition-transform;
+.grid-in-to {
+  grid-area: to;
 }
 
-.markdown-content details {
-  @apply bg-warning text-warning-content p-3 rounded-md;
+.grid-in-thumbnail {
+  grid-area: thumbnail;
 }
 
-.markdown-content details[open] > summary {
-  @apply mb-2;
+.grid-in-actions {
+  grid-area: actions;
 }
 
-.markdown-content details[open] > summary::after {
-  @apply rotate-180;
+.markdown-content {
```

Comment:
CSS natively supports nesting on all major browsers now. I could've done this before the TailwindCSS v4 upgrade, but I didn't knoow about the feature at the time.

Reply:
Me either, nice.